wagtail/wagtailadmin/templates/wagtailadmin/notifications/base_notification.html

```diff
@@ -1,3 +1,4 @@
-{% load i18n %}{% block notification %}{% endblock %}
-
+{% load i18n %}{% block subject %}{% endblock %}
+{% block greeting %}{% trans "Hello" %} {% if user.first_name %}{{ user.first_name }}{% else %}{{ user.username }}{% endif %},{% endblock %}
```

Comment:
This will behave badly for users with a custom User model that does not have a first_name and/or username field. Using an email address instead of a username is a fairly common modification. You can use the `User.get_short_name()` and `User.get_username()` methods instead. They will act the same for the default User model, but custom User models can override them with something more relevant.